inc/functions.inc.php

```diff
@@ -44,6 +44,85 @@ function st_register_widget() {
     }
 }
 
+function tp_disp_boolean($bool_text)
+{
+    $bool_text = (string)$bool_text;
+    if (empty($bool_text) || '0' === $bool_text || 'false' === $bool_text) {
+        return 'false';
+    }
+
+    return 'true';
+}
+
+/**
+ * Add custom filter on edit posts page
+ * 
+ * Filter posts by year
+ */
+
+//var_dump(get_option('taxopress_external_taxonomies')['category']['show_filters']);
```

Comment:
`//var_dump(get_option('taxopress_external_taxonomies')['category']['show_filters']);`

Leaving debugging code in final commit is discouraged. After var_dump or doing any debug, always make sure it's removed before making your commit to development.

So, remove every instance of `var_dump` in your commit